Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -278,7 +278,7 @@ final class ConfigGenerator: ConfigGenerating {
         if target.product == .unitTests {
             settings["TEST_HOST"] =
                 .string(
-                    "$(BUILT_PRODUCTS_DIR)/\(app.target.productNameWithExtension)/$(BUNDLE_EXECUTABLE_FOLDER_PATH)/\(app.target.productName)"
+                    "$(BUILT_PRODUCTS_DIR)/\(app.target.productNameWithExtension)/$(BUNDLE_EXECUTABLE_FOLDER_PATH)\(app.target.productName)"
```

Comment:
What about using `RelativePath` to concatenate all the components and then turning them into a `String`. Otherwise, the same issue might happen in the future if any of the variables introduces a forward slash suffix.

Reply:
will try